### ISSUE TITLE
Missing /dev/shm

### ISSUE THREAD

**Issue by yesimxev** [offensive-security/kali-nethunter#1265]
### Device:
Oneplus One, S6 edge
### OS version (KitKat/Lollipop/Marshmallow/Nougat):
Marshmallow, Nougat
### Built from repo (date and build command) or downloaded from website (links):
Github
### Output of `cat /proc/version` in adb shell or NetHunter Terminal:
Linux version 3.10.61-yesimxev-gab2e937-dirty (root@kali) (gcc version 4.9.x-google 20140827 (prerelease) (GCC) ) #1 SMP PREEMPT Fri Mar 1 00:03:43 CET 2019 
## Issue:
Not sure what is it related to, but realised in the last few months fs build, in the nethunter VNC session. The firefox has a blank page, no menus, no webpage etc, unless /dev/shm exists. So when I do 
`mkdir /dev/shm`
it all works fine as before, webpage pops in straightaway.

**Comment by jcadduono** [comment]
does your custom kernel have CONFIG_SHMEM=Y?

**Comment by yesimxev** [comment]
Yes, it is enabled in my zerolte and probably in binkybear's Oneplus One kernel source too, can't check that one at the moment

**Comment by jcadduono** [comment]
hmm, i wonder if we need to make an shm mount in the bootkali_init script https://github.com/offensive-security/nethunter-app/blob/master/assets/scripts/bootkali_init

**Comment by yesimxev** [comment]
Shall we add the mkdir command to the init? Btw would be good to know if it happens on other devices too. The command was not needed a few months ago though

**Comment by Re4son** [comment]
Should be fixed in https://github.com/offensive-security/nethunter-app/commit/78a3fd683c3cbaebe51775f3541d58a2d4ba59be

Could you give it another try using the 2019.2 branch please?
